docs/content/integrations/dbt/reference.mdx

```diff
@@ -53,10 +54,10 @@ For a step-by-step implementation walkthrough, refer to the [Using dbt with Dags
   to see this concept in context.
 </Note>
 
-You can create a Dagster project that wraps your dbt project by using the [`dagster-dbt project scaffold`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold) command line interface.
+You can create a Dagster project that wraps your dbt project by using the [`dagster-dbt project scaffold`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold) command line interface. We recommend using the `--use-dbt-project` flag to use <PyObject object="DbtProject" module="dagster_dbt" />.
```

Comment:
Since we're standardizing on `DbtProject`, I think we should omit the last sentence. We shouldn't need to call attention to the fact that we're doing something special here, especially since we're planning on updating the tutorial.

We should probably just make `DbtProject` opt-out so that they don't even need to pass in a flag in the first place.

[1]

```suggestion
You can create a Dagster project that wraps your dbt project by using the [`dagster-dbt project scaffold`](/\_apidocs/libraries/dagster-dbt#dagster-dbt-project-scaffold) command line interface.

Reply:
> We should probably just make DbtProject opt-out so that they don't even need to pass in a flag in the first place.

I agree - that's the plan actually. Once both the tutorial and the dbt course in Dagster University are updated, I will refactor the CLI and the scaffold template to remove the flag and make the use of `DbtProject` the default and only possible strategy.  Then I'll do a second pass on the doc to remove the `flag` - everything else will be already up-to-date.

Reply:
All changes made in [6ed3208](https://github.com/dagster-io/dagster/pull/23012/commits/6ed3208f238880c0ba1c6e3a73bc854c2efcff28)